source/win32_tinyengine.c

```diff
@@ -5,6 +5,234 @@
 
 #include "tinyengine_types.h"
 
+typedef struct win32_digital_button win32_digital_button;
+struct win32_digital_button
```

Comment:
Why is this platform specific it looks like all platforms could probably use this structure for their buttons

Reply:
I definitely would prefer the way you suggest, but I didn't know how to implement that given the nature of `Win32MainWindowCallback()` and `WM_INPUT`. See:
https://discordapp.com/channels/613078025386459307/674441831664058407/675846106667810850